src/rt/sched/cown.h

```diff
@@ -744,8 +751,10 @@ namespace verona::rt
     static void schedule(size_t count, Cown** cowns, Args&&... args)
     {
       static_assert(std::is_base_of_v<Behaviour, Be>);
+#ifdef USE_SYSTEMATIC_TESTING
```

Comment:
What is the purpose of this log message being excluded outside of systematic testing?

Reply:
At some point I tried to remove the dependency on typeid, but it turned out that it is not important to do so. Thank you very much for pointing it it, removed it.